srcCpp/Infrastructure_pro.cxx

```diff
@@ -539,8 +539,8 @@ bool configureShmemTransport(
     int retcode;
     int key = rand();
     int minBufferSize = parentMsgSizeMax;
-    int step = 1048576 + parentMsgSizeMax; // 1MB + parentMsgSizeMax
-    int maxBufferSize = 60817408; // 58MB
+    int step = 1048576; // 1MB
+    int maxBufferSize = (std::max)(60817408 /* 58MB */, parentMsgSizeMax);
```

Comment:
I need to find out why we choose 58MB, this might look a bit random, and might need to be changed or explained. @fercsrti77, any feedback on this?